contributing.rst

```diff
@@ -201,6 +201,13 @@ Android Studio
 
 On Mac, add the following to your :file:`.bash_profile`
 
+.. note::
+    On Windows, you have to run Andriod Studio once to complete the Installation of ADB. The tool can be found in ``C:\Users\your user name\AppData\Local\Android\sdk\platform-tools``. You need to add it to the environment variable path, use the following command:
```

Comment:
s/Andriod/Android

Use the `:file:` text role for file paths

Reply:
Done.